docs/installation.rst

```diff
@@ -4,9 +4,17 @@ Installation
 
 At the command line::
 
+    $ pip install dateparser
+
+Or, if you don't have pip installed::
+
     $ easy_install dateparser
 
-Or, if you have virtualenvwrapper installed::
+If you want to install from the latest sources, you can do::
 
-    $ mkvirtualenv dateparser
-    $ pip install dateparser
+    $ git clone https://github.com/scrapinghub/dateparser.git
+    $ cd dateparser
+    $ python setup.py install
+
+Depending on how your environment is set up, you might need to use sudo for the
```

Comment:
Let's not recommend sudo over creating virtual environment. I think our users are smart enough to figure differences on their own, and it would not be the first  module they install :)